src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -97,10 +97,9 @@ const Stream = createReactClass({
 
     this.fetchSavedSearches();
     this.fetchProcessingIssues();
-  },
-
-  componentDidMount() {
-    this.fetchData();
+    if (!this.state.loading) {
+      this.fetchData();
```

Comment:
Do we know why this was in a separate lifecycle method? Is it possible that there are some dependency issues between the different fetch methods that were covered up by this being in separate lifecycles?

Also cDM is generally the recommended place to do data fetching.

Reply:
Yeah, i added it there, but not for any particular reason. Just grouping them all